app/overrides/spree/admin/properties/index.rb

```diff
@@ -0,0 +1,26 @@
+Deface::Override.new(
+  virtual_path: 'spree/admin/properties/index',
+  name: 'remove_conditions',
+  remove: "erb[silent]:contains('if current_spree_user.respond_to?(:has_spree_role?) && current_spree_user.has_spree_role?(:admin)')"
+)
+
+Deface::Override.new(
```

Comment:
@pmilewski qq on this change: could you please give me some more context on why this change is needed? This is causing a syntax error on my end because an `if` block is inserted but doesn't have an `end` close block.

![Screen Shot 2019-12-26 at 8 35 17 PM](https://user-images.githubusercontent.com/7424043/71501422-48e0a300-281f-11ea-9e0f-0df43994e034.png)

I'm not sure how Deface works though. When I uncommented line 7 to 26, everything works fine both on admin and vendor views.